### DISCUSSION TITLE
Not be able to ping the monitoring interface for SO, and no detect events and alerts

### DISCUSSION THREAD

**Discussion by habibbarika** [Security-Onion-Solutions/securityonion#11325] [General]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/9c8996f3-6027-4d2f-bceb-4b302ea5087e)

I am not able to ping from Linux1 or Linux2 to the security onion interface. And no detect alert for arpspoof attack from Linux1 to Linux2. I used VMWare workstation with Bridge and Vmnet1 (Host-only), as given in the figure.

**Comment by Mav1814** [comment]
Hi, 
It's normal not to be able to ping Security Onion. When you ping it, it says Destination Host Prohibited.

**Reply by jwmasekre** [reply]
Specifically, your monitoring interface shouldn't have an IP address to ping anyways.

As far as no traffic, I believe you have to do some extra configurations, including creating a vswitch, to get traffic to that port. I'll follow this up with instructions once I find them

**Reply by habibbarika** [reply]
Creation of virtual switch not existe in  VMWare workstation, but in Esxi

**Comment by habibbarika** [comment]
This is my configuration 👍 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/96f62a0f-ec73-4e5e-bdc7-a24673a0daa7)

Thanks lot !

**Comment by habibbarika** [comment]
Is there a command to check/modify the monitoring interface?

**Reply by dougburks** [reply]
You ran `ifconfig ens35` in your screenshot above. You can also run `ifconfig bond0`. Additionally, you can run tcpdump on bond0 and/or ens35 to verify that they are seeing the traffic that you expect them to see.

**Comment by habibbarika** [comment]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/620c60e2-4866-42d7-8a38-694da4ed2af5)

**Comment by habibbarika** [comment]
I have new installation ens36 instead ens35.
Arpspoof attack ; 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/e47a6250-3ffe-43bd-b09a-8adeda107b86)

Now i receive traffic : 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/aee74dbb-a2bb-4a87-bf89-0f4f0cff0532)
 But the SOC no detection (Alert, Hunt)

Thanks

**Comment by habibbarika** [comment]
SOC 👍 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/cca61c9a-c442-46e9-8164-053e10dc8e1b)

**Reply by habibbarika** [reply]
With bond0 interface 👍 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/9bf9ddf1-4ef3-43ca-8c73-df865f55b782)

**Comment by dougburks** [comment]
Try creating traffic that would match your enabled NIDS rules in /opt/so/rules/nids/all.rules.

For example:
https://docs.securityonion.net/en/2.4/managing-alerts.html#nids-testing

**Reply by habibbarika** [reply]
Thank you very much Doug!

actually, there is no rule activated for arpspoof (It doesn't even exist). Your answer helped me discover the meerkat rules (How to enable/disable the rules). I use so 2.4.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/1f85789c-5369-4bff-903d-76af11b5438a)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/e54d816b-f956-4eb1-bf96-b461e2e364b5)

It's the only this rule does match 👍 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/120285406/e4f080ec-8079-42a1-95a1-8b99386931e7)

If you can help me create a rule for arpspoof or DoS (with Hping3). At least direct me to a site or link.

thanks again !

**Comment by habibbarika** [comment]
Arp attacks related to layer 2 of OSI model. Suricata and Snort IDPS is developed to detect attacks at the higher level of OSI model. Suricata doesn't have mechanism to detect such type attacks.
